app/packages/looker/src/worker/shared.ts

```diff
@@ -1,5 +1,12 @@
 import { HEATMAP } from "@fiftyone/utilities";
 
+export type DenseLabelRenderStatus =
+  | null
+  | "pending"
+  | "painting"
+  | "painted"
+  | "decoded";
```

Comment:
These statuses are used quite frequently. Something like the below could be useful
```suggestion
export const DECODED = "decoded";
export const PAINTED = "painted";
export const PAINTING = "painting";
export const PENDING = "pending";
export type DenseLabelRenderStatus =
  | null
  | DECODED
  | PENDING
  | PAINTED
  | PAINTING
```